extensions/kube-object-event-status/package.json

```diff
@@ -18,8 +18,8 @@
   "dependencies": {},
   "devDependencies": {
     "@k8slens/extensions": "file:../../src/extensions/npm/extensions",
-    "ts-loader": "^8.0.4",
-    "typescript": "^4.3.2",
-    "webpack": "^4.46.0"
+    "ts-loader": "latest",
```

Comment:
We shouldn't use `latest` here because it will break if there is a mismatch of versions between this file and `../../package.json`

Reply:
Why it will break when it's not used in "production" / inside packed extension?
I thing only `ts-loader` must be compatible with local `typescript` package.

Reply:
We do run `npm install` on extensions, even bundled extensions.

Reply:
When we do? So maybe we should move it to `peerDependencies` then?

Reply:
@Nokel81 @ixrock Lens installs extension npm package without development dependencies -> extension dev dependencies don't need to match same versions on Lens side.

Reply:
Note: using `latest` version might still be a bit risky (in general).

Reply:
I think it's not more risky than just having outdated specific version == in both cases I assume it's easy to spot/fix/use what is needed per extension package.